src/Client.php

```diff
@@ -17,18 +17,11 @@ class Client {
   const QUERY_COMPLETE = 4;
   const QUERY_JOB_CANCELLED = 5;
 
-  private $queryStatusDefinitions = [
```

Comment:
Removing this doesn't seem related?

Reply:
PHPstorm just complained that this var is not being used at all.